source/default_mode/Symbiont.h

```diff
@@ -561,7 +573,16 @@ class Symbiont: public Organism {
         //points = points - my_config->SYM_HORIZ_TRANS_RES();
         SetPoints(0);
         emp::Ptr<Organism> sym_baby = Reproduce();
-        my_world->SymDoBirth(sym_baby, location);
+        emp::WorldPosition new_pos = my_world->SymDoBirth(sym_baby, location);
+
+        //horizontal transmission data nodes
+        emp::DataMonitor<int>& data_node_attempts_horiztrans = my_world->GetHorizontalTransmissionAttemptCount();
```

Comment:
Ditto here, do we need to call the Get methods each time or can we use the instance variables when we're sure it has been created?